docs/reference/release-notes.md

```diff
@@ -1,16 +1,16 @@
 ---
 id: release-notes
 title: "Release notes"
-description: "Release notes for Camunda Platform 8 and its components."
+description: "Release notes for Camunda 8 and its components."
 ---
 
-Release notes for Camunda Platform 8, including alphas, are available on [GitHub](https://github.com/camunda/camunda-platform/releases). This includes release assets and release notes for Zeebe, Operate, Tasklist, and Identity.
+Release notes for Camunda 8, including alphas, are available on [GitHub](https://github.com/camunda/camunda-platform/releases). This includes release assets and release notes for Zeebe, Operate, Tasklist, and Identity.
 
 The current release notes can be found [here](https://github.com/camunda/camunda-platform/releases/latest).
 
 [Update guides](/self-managed/operational-guides/update-guide/introduction.md) include links to both release notes and release blogs.
 
-Most releases and release notes are available through the [Camunda Platform repo](https://github.com/camunda/camunda-platform), however, the following component release notes are available as linked below:
+Most releases and release notes are available through the [Camunda repo](https://github.com/camunda/camunda-platform), however, the following component release notes are available as linked below:
```

Comment:
This may still need to be the "Camunda Platform repo".

Reply:
More context here - https://github.com/camunda/developer-experience/issues/122#issuecomment-1734331485

Reply:
Adjusted in commit 👍